### ISSUE TITLE
vpn server in same region

### ISSUE THREAD

**Issue by Reshad-Hasan** [outlandnish/poc-vpn-detection#2]
If there is a VPN server in my city then, won't the VPN software select that server as it is closest every time? In that case, the server and browser timezone will be the same always.

**Comment by outlandnish** [comment]
Yeah, this was just a silly idea I had before going to bed. The proper way to do VPN detection would be by blacklisting IP addresses of known VPN servers. I just thought it would be fun to implement!
